<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F√ºtterung Abgeschlossen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    ;
    
     &quot;Segoe UI&quot;, Arial, sans-serif;
}

QLabel {
     16px;
    padding: 8px;
}

QPushButton {
    
    color: white;
     18px;
     bold;
    padding: 12px 24px;
    min-height: 60px;
}

.title-label {
     32px;
     bold;
    
    padding: 20px;
}

.summary-label {
     24px;
     bold;
    
    padding: 16px;
}

.info-label {
     20px;
    
    padding: 12px;
}

.value-label {
     22px;
     bold;
    
    padding: 12px;
}</string>
  </property>
  <widget class="QLabel" name="label_titel">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>80</y>
     <width>924</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string> 32px;  bold</string>
   </property>
   <property name="text">
    <string notr="true">F√ºtterung erfolgreich abgeschlossen!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_zusammenfassung">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>180</y>
     <width>824</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame { }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <widget class="QLabel" name="label_gesamtmenge_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 20px</string>
    </property>
    <property name="text">
     <string>Gesamtmenge gef√ºttert:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gesamtmenge_wert">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 22px;  bold</string>
    </property>
    <property name="text">
     <string>135.0 kg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_futtertyp_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 20px</string>
    </property>
    <property name="text">
     <string>Futtertyp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_futtertyp_wert">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 22px;  bold</string>
    </property>
    <property name="text">
     <string>Heu eigen 2025</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pferde_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 20px</string>
    </property>
    <property name="text">
     <string>Pferde gef√ºttert:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pferde_wert">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 22px;  bold</string>
    </property>
    <property name="text">
     <string>30 von 32 Boxen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dauer_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 20px</string>
    </property>
    <property name="text">
     <string>F√ºtterungsdauer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dauer_wert">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 22px;  bold</string>
    </property>
    <property name="text">
     <string>45 Minuten</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_naechste_text">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 20px</string>
    </property>
    <property name="text">
     <string>N√§chste F√ºtterung:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_naechste_wert">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string> 22px;  bold</string>
    </property>
    <property name="text">
     <string>17:30 Uhr</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_neue_fuetterung">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>530</y>
     <width>300</width>
     <height>84</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string></string>
   </property>
   <property name="text">
    <string>üîÑ Neue F√ºtterung</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zum_start">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>530</y>
     <width>300</width>
     <height>84</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string></string>
   </property>
   <property name="text">
    <string>üè† Zum Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
